python/GafferSceneUI/LightEditor.py

```diff
@@ -319,6 +320,57 @@ def __buttonDoubleClick( self, pathListing, event ) :
 
 		return True
 
+	def __keyPress( self, pathListing, event ) :
+
+		if event.key == "Return" or event.key == "Enter" :
+
+			selection = pathListing.getSelection()
+			columns = pathListing.getColumns()
+
+			inspections = []
+
+			with Gaffer.Context( self.getContext() ) as context :
+				for i in range( 0, len( columns ) ) :
+					column = columns[ i ]
+					if not isinstance( column, _GafferSceneUI._LightEditorInspectorColumn ) :
+						continue
+					for path in selection[i].paths() :
+						context["scene:path"] = GafferScene.ScenePlug.stringToPath( path )
+						inspection = column.inspector().inspect()
+
+						if inspection is not None :
+							inspections.append( inspection )
+
+			nonEditable = [ i for i in inspections if not i.editable() ]
+
+			if len( nonEditable ) == 0 :
+				edits = [ i.acquireEdit() for i in inspections ]
+				warnings = "\n".join( [ i.editWarning() for i in inspections if i.editWarning() != "" ] )
+
+				self.__popup = GafferUI.PlugPopup( edits, warning = warnings )
+
+				if isinstance( self.__popup.plugValueWidget(), GafferSceneUI.TweakPlugValueWidget ) :
+					self.__popup.plugValueWidget().setNameVisible( False )
+
+				self.__popup.popup()
+			else :
+
+				# See todo in `PlugPopup._PopupWindow`
+				PopupWindow = GafferUI.PlugPopup.__bases__[0]
```

Comment:
Note to self : this is all sorted on `main` now, so you'll need to sort it during the merge.